demos/integration/lmpid-signal-gpt.html.tpl

```diff
@@ -184,7 +191,15 @@ e:5d6d6ed5354f68d7523b7b39330145346209d20b06f5ed32373583823bac8d1a</code></pre>
           googletag.display('div-gpt-ad-1682350431454-0');
           googletag.display('div-gpt-ad-1682350702718-0');
           googletag.display('div-gpt-ad-1682350744052-0');
-        });
+        })
+
+        googletag.cmd.push(function () {
+          optable.cmd.push(function () {
+            optable.instance.targeting().then(function() {
+              googletag.pubads().refresh();
```

Comment:
Note in the actual code ran in the page we do wait on targeting data to be populated before refreshing ads (the alternative documented in the example) to make sure we include the lmpid even when no cache is populated yet